server/chat-commands/moderation.ts

```diff
@@ -1206,21 +1256,21 @@ exports.commands = {
 	globaldeclare(target, room, user) {
 		if (!target) return this.parse('/help globaldeclare');
 		if (!this.can('gdeclare')) return false;
-		target = this.canHTML(target);
+		(target as string | null) = this.canHTML(target);
 		if (!target) return;
 
 		for (const u of Users.users.values()) {
 			if (u.connected) u.send(`|pm|~|${u.group}${u.name}|/raw <div class="broadcast-blue"><b>${target}</b></div>`);
 		}
 		this.modlog(`GLOBALDECLARE`, null, target);
 	},
-	globaldeclarehelp: [`/globaldeclare [message] - Anonymously announces a message to every room on the server. Requires: ~`],
+	globaldeclarehelp: [`/globaldeclare [message] - Anonymously announces a message to all rooms on the site. Requires: ~`],
 
 	cdeclare: 'chatdeclare',
 	chatdeclare(target, room, user) {
 		if (!target) return this.parse('/help chatdeclare');
 		if (!this.can('gdeclare')) return false;
-		target = this.canHTML(target);
+		(target as string | null) = this.canHTML(target) || null;
```

Comment:
```suggestion
		(target as string | false) = this.canHTML(target);
```
The left-hand side should just match `canHTML`'s signature... which I think it already does?